.travis.yml

```diff
@@ -0,0 +1,14 @@
+os:
+  - linux
+sudo: required
```

Comment:
Is this needed? It's possible to install apt dependencies with `addons.apt`: https://docs.travis-ci.com/user/installing-dependencies/#Adding-APT-Packages
(builds without `sudo` are a lot faster)

Reply:
@felixfbecker, thank you for review. Your proposal looks reasonable. Please prepare an update and request a pull request to the branch  [Dev4-1_PSCore_compatibility](https://github.com/pester/Pester/tree/Dev4-1_PSCore_compatibility).

Reply:
@felixfbecker thanks for that, I did not really know what I am doing when I was putting the travis.yml in place